### PR TITLE
chore(deps): Update aap, keycloak and ocm backends

### LINKED ISSUE #756: Update ocm, aap and Keycloak versions
Plugins should be upgraded to a version that includes https://github.com/janus-idp/backstage-plugins/pull/827 to make sure that users can configure the scheduler in the app-config.yaml

### PR DESCRIPTION
**Pull request by jerolimov**
## Description

Update dynamic-plugin peer dependencies to ~~add 3scale and~~ update AAP, Keycloak, and OCM backends to include these two PRs:

* https://github.com/janus-idp/backstage-plugins/pull/938
* https://github.com/janus-idp/backstage-plugins/pull/945

## Which issue(s) does this PR fix

* Fixes #756 to include https://github.com/janus-idp/backstage-plugins/issues/629

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by jerolimov** [comment]
@Zaperex @kadel @schultzp2020 regarding https://github.com/janus-idp/backstage-showcase/issues/756#issuecomment-1806102755

Should I add 3scale similar to the other dynamic plugins? Is there anything else to do or to verify?

**Comment by Zaperex** [comment]
3scale wasn't originally installed statically in the showcase, so I'm not sure if we should be including it dynamically as well. If we do want to include it, I think we'll want to include some default configurations in https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins.default.yaml and https://github.com/janus-idp/backstage-showcase/blob/main/app-config.dynamic-plugins.yaml.

**Review by schultzp2020 — APPROVED** [review]
LGTM

**Review by davidfestal — CHANGES_REQUESTED** [review]
I think @Zaperex remark [here](https://github.com/janus-idp/backstage-showcase/pull/798#issuecomment-1819439746) is right.

Could we split the PR, and keep the addition of the 3Scale plugin for a distinct PR ?

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -16,13 +16,14 @@
     "tar": "6.2.0"
   },
   "peerDependencies": {
-    "@janus-idp/backstage-plugin-aap-backend": "1.4.3",
+    "@janus-idp/backstage-plugin-3scale-backend": "1.3.4",
```
Comment (tumido) [inline dynamic-plugins/imports/package.json:19]: This plugin is missing configuration in `app-config.dynamic-plugins.yaml` and `dynamic-plugins.default.yaml`

**Review by tumido — CHANGES_REQUESTED** [review]

**Comment by tumido** [comment]
Also an important question - should 3scale even be included? Adding it into the base image we make it a supported plugin (no matter if disabled by default or not). Do we have a formal decision that 3scale is a supported plugin?

**Comment by jerolimov** [comment]
Let me remove 3scale for now. :)

**Comment by tumido** [comment]
As per https://redhat-internal.slack.com/archives/C05HGAR2DT5/p1700514326249259?thread_ts=1700250550.958579&cid=C05HGAR2DT5

3scale is not a supported plugin, therefore it should not be included in the image.

**Comment by jerolimov** [comment]
I reverted this PR to my initial commit that just updates aap, keycloak and ocm backends.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Review by davidfestal — APPROVED** [review]

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**